SU-SOE/soe_basic
# D8CORE-3486: Correcting people link colors and updated Decanter

# READY FOR REVIEW

# Summary
- Corrected orange link hover colors
- Updated Decanter version

# Review By (Date)
- 2/9

# Urgency
- Low

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Verify the orange links on the right col of a person page have a white hover.

# Affected Projects or Products
- SOE Person

# Associated Issues and/or People
- D8CORE-3486

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
